src/main/java/ch/njol/skript/SkriptCommand.java

```diff
@@ -446,6 +447,15 @@ else if (args[0].equalsIgnoreCase("test") && TestMode.DEV_MODE) {
 					);
 			}
 
+			else if (args[0].equalsIgnoreCase("list") || args[0].equalsIgnoreCase("show")) {
+				info(sender, "list.enabled", StringUtils.join(ScriptLoader.getLoadedScripts().stream()
+						.map(script -> script.getConfig().getFileName())
+						.toArray(String[]::new)));
+				info(sender, "list.disabled", StringUtils.join(ScriptLoader.getDisabledScripts().stream()
```

Comment:
I think the message should include the path of the script as users can have scripts of the same name in different folders